gallery/src/components/NxBinaryDonutChart/NxBinaryDonutChartMinimalExample.tsx

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxBinaryDonutChart } from '@sonatype/react-shared-components';
+
+export default function NxBinaryDonutChartMinimalExample() {
+  return <NxBinaryDonutChart percent={90} />;
```

Comment:
You'll notice that in my branch I added more examples at different percentages.  Keep in mind that the gallery actually has (at least) two purposes:  not just to demonstrate to consumers how to use the components, but also to prove to ourselves that the styles (and logic) work correctly.  Therefore I do think you should add donuts at a variety of percentages to at least some of the examples.  And especially covering the edge cases.

Reply:
> You'll notice that in my branch I added more examples at different percentages. Keep in mind that the gallery actually has (at least) two purposes: not just to demonstrate to consumers how to use the components, but also to prove to ourselves that the styles (and logic) work correctly. Therefore I do think you should add donuts at a variety of percentages to at least some of the examples. And especially covering the edge cases.

I added three more examples ( 25, 50, 100), I can add more if needed e8460440bc68f3dd35da44cd427a523a3659fa99

Reply:
> I added three more examples ( 25, 50, 100), I can add more if needed e846044

Two things:
1. I don't think each one needs to be a separate example.  You can have a single example for each "style" (e.g. each `innerRadius`) and within that example have a series of donut charts with varying percentages.  That would keep the verbosity of the page down.
2. I think we definitely also need one for 0%

Reply:
> > I added three more examples ( 25, 50, 100), I can add more if needed [e846044](https://github.com/sonatype/sonatype-react-shared-components/commit/e8460440bc68f3dd35da44cd427a523a3659fa99)
> 
> Two things:
> 
> 1. I don't think each one needs to be a separate example.  You can have a single example for each "style" (e.g. each `innerRadius`) and within that example have a series of donut charts with varying percentages.  That would keep the verbosity of the page down.
> 2. I think we definitely also need one for 0%

Done b9f8078b190bac0c253d7a7c363ad492ea6a5587